File: vdp-php/tests/vctc/ValidationTest.php
<?php

namespace Vdp;

class ValidationTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->visaAPIClient = new VisaAPIClient;
	}
	
	public function testRetreiveListofDecisionRecords() {
		$baseUrl = "vctc/";
		$resourcePath = "validation/v1/decisions/history";
		$queryParams = "?limit=1&page=1";
		$statusCode = $this->visaAPIClient->doMutualAuthCall ( 'get', $baseUrl.$resourcePath.$queryParams, 'Retrieve List of Recent Decision Records call', '' );
		$this->assertEquals($statusCode, "200");
	}
}

File: vdp-c-sharp/vdp-c-sharp/VDPApiTest/vctc/ValidationTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Vdp
{
    [TestClass]
    public class ValidationTest
    {
        private VisaAPIClient visaAPIClient;

        public ValidationTest()
        {
            visaAPIClient = new VisaAPIClient();
        }

        [TestMethod]
        public void TestRetreiveListofDecisionRecords()
        {
            string baseUri = "vctc/";
            string resourcePath = "validation/v1/decisions/history";
            string queryString = "?limit=1&page=1";
            string status = visaAPIClient.DoMutualAuthCall(baseUri + resourcePath + queryString, "GET", "Retreive List of Decision Records Test", "");
            Assert.AreEqual(status, "OK");
        }
    }
}
